oak_runtime/introspection_browser_client/components/ObjectAsTree/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import JSONTree from 'react-json-tree';
+
+const THEME = {
```

Comment:
Was this taken from somewhere? worth referencing the source, in case someone wanted to change / expand it later?

Reply:
Good point, added :)